README.md

```diff
@@ -1,3 +1,17 @@
+---
+
+This library, ADAL for .NET, will no longer receive new feature improvements. Instead, use the new library
+[MSAL for .NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
```

Comment:
nit: maybe add here "....which integrates with the Microsoft identity platform" ...?

Reply:
Trying to keep it stylistically consistent, so not going to make that change here.